packages/salesforcedx-vscode-core/src/commands/viewChanges.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {
+  LibraryCommandletExecutor,
+  SourceTrackingService
+} from '@salesforce/salesforcedx-utils-vscode';
+import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode/src/types';
+import { channelService, OUTPUT_CHANNEL } from '../channels';
+import { nls } from '../messages';
+import {
+  EmptyParametersGatherer,
+  SfdxCommandlet,
+  SfdxWorkspaceChecker
+} from './util';
+
+export class SourceTrackingGetStatusExecutor extends LibraryCommandletExecutor<
```

Comment:
This one does seem net-new for us, so we probably would want to create tests here.